client/src/components/_common/__tests__/TreeItem.test.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { Router } from 'react-router';
+import TreeView from '@mui/lab/TreeView';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import TreeItemLabel from 'components/TestSuite/TestSuiteTree/TreeItemLabel';
+import ThemeProvider from 'components/ThemeProvider';
+import { createMemoryHistory } from 'history';
+import CustomTreeItem from '../TreeItem';
+import { mockedTestSuite } from '../__mocked_data__/mockData';
+
+const runTestsMock = jest.fn();
+
+test('renders custom TreeItem', () => {
+  render(
+    <ThemeProvider>
+      <TreeView>
+        <CustomTreeItem
+          nodeId={mockedTestSuite.id}
+          label={
+            <TreeItemLabel
+              runnable={mockedTestSuite}
+              runTests={runTestsMock}
+              testRunInProgress={false}
+            />
+          }
+          // eslint-disable-next-line max-len
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-explicit-any
```

Comment:
Is it worth just configuring eslint to ignore these rules in test folders (I'm assuming eslint lets you do that)?

Reply:
That might be worth doing!